### PR TITLE
When the user did not added any address on its account, change wording to "Add" instead of "Edit" in the link

### PR DESCRIPTION
**Pull request by Mahjouba91**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

It makes more sense to have a correct action wording, instead of using "Edit" whereas there is not any address yet.

### CODE REVIEW

File: templates/myaccount/my-address.php [hunk 1]
```diff
@@ -44,15 +44,20 @@
...
-<?php foreach ( $get_addresses as $name => $title ) : ?>
+<?php foreach ( $get_addresses as $name => $title ) : 
+		
+	$address = wc_get_account_formatted_address( $name ); ?>
 
 	<div class="u-column<?php echo ( ( $col = $col * -1 ) < 0 ) ? 1 : 2; ?> col-<?php echo ( ( $oldcol = $oldcol * -1 ) < 0 ) ? 1 : 2; ?> woocommerce-Address">
 		<header class="woocommerce-Address-title title">
 			<h3><?php echo $title; ?></h3>
-			<a href="<?php echo esc_url( wc_get_endpoint_url( 'edit-address', $name ) ); ?>" class="edit"><?php _e( 'Edit', 'woocommerce' ); ?></a>
+			<?php if ( $address ) : ?>
```
Comment (claudiosanches) [inline templates/myaccount/my-address.php]: We could save a few lines of code, replacing:

```php
<?php if ( $address ) : ?>
    <a href="<?php echo esc_url( wc_get_endpoint_url( 'edit-address', $name ) ); ?>" class="edit"><?php _e( 'Edit', 'woocommerce' ); ?></a>
<?php else: ?>
    <a href="<?php echo esc_url( wc_get_endpoint_url( 'edit-address', $name ) ); ?>" class="edit"><?php _e( 'Add', 'woocommerce' ); ?></a>
<?php endif; ?>
```
With:
```php
<a href="<?php echo esc_url( wc_get_endpoint_url( 'edit-address', $name ) ); ?>" class="edit"><?php echo $address ? esc_html__( 'Edit', 'woocommerce' ) : esc_html__( 'Add', 'woocommerce' ); ?></a>
```

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Thanks for your help!
Just one small change to be made and then I'll be good to go.

**Comment by Mahjouba91** [comment]
Hey, Thanks for your review, I have applied it.

**Review by claudiosanches — APPROVED** [review]
Looks great, thanks for your help!

**Pull request merged**